gui/static/js/components/controlpanel.js

```diff
@@ -285,8 +347,8 @@ export default class ControlPanel extends React.Component {
                     <tr>
                         <td>
                             <label>
-                                 Choose bot:
-                                <select value={this.props.currentBot} onChange={this.selectBot}> id="botlist" name="bots">
+                                Choose bot:
+                                <select value={this.state.currentBot} onChange={this.selectBot} id="botlist" name="bots">
```

Comment:
Please make sure this is a correct reference (and that you haven't changed this by accident).